scripts/ruby_common/daily_snapshot.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require_relative 'slack_client'
+require_relative 'docker_utils'
+require 'rubygems'
+require 'logger'
+require 'fileutils'
+require 'active_support/time'
```

Comment:
It's quite a big one, coming with Ruby on Rails. What's stopping us from using standard time?

Reply:
Apparently it is required by `sleep(1.hour)`. Is there a way to sleep for an hour without using that gem?